openedx/features/survey_report/application.py

```diff
@@ -0,0 +1,35 @@
+"""
+Contains the logic to manage survey report model.
+"""
+
+from django.conf import settings
+
+from openedx.features.survey_report.models import SurveyReport
+from openedx.features.survey_report.queries import (
+    get_course_enrollments,
+    get_currently_learners,
+    get_generated_certificates,
+    get_learners_registered,
+    get_unique_courses_offered
+)
+
+
+def generate_report() -> None:
```

Comment:
[OEP-49 conventions](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0049-django-app-patterns.html#api-py) suggest to call this module `api.py` instead of `application.py`.

Reply:
@ormsbee understood, I changed the file name.